src/smc-webapp/course/handouts_panel.cjsx

```diff
@@ -0,0 +1,563 @@
+# SMC libraries
+misc = require('smc-util/misc')
+{defaults, required} = misc
+{salvus_client} = require('../salvus_client')
+
+# React Libraries
+{React, rclass, rtypes} = require('../smc-react')
+{Alert, Button, ButtonToolbar, ButtonGroup, Input, Row, Col, Panel, Table} = require('react-bootstrap')
+
+# SMC and course components
+course_funcs = require('./course_funcs')
+styles = require('./styles')
+{BigTime, FoldersToolbar} = require('./common')
+{Icon, Tip, SearchInput, MarkdownInput} = require('../r_misc')
+
+# Could be merged with steps system of assignments.
+# Probably not a good idea mixing the two.
+# Could also be coded into the components below but steps could be added in the future?
```

Comment:
I don't want to mix the two given the express difference in concept between handouts and assignments even though both are just folders. 

In addition to the reason given, structuring the code this way also provides a nice stepping stone for future devs to understand what's happening in the assignments' `STEPS` related code.